svc-api/main.go

```diff
@@ -151,11 +152,24 @@ func main() {
 		log.Fatal(err.Error())
 	}
 
+	// TODO: uncomment the following line after the migration
+	config.CollectCLArgs()
+
+	// TODO: remove the InitializeService for GoMicro after the migration
 	err = services.InitializeService(services.APIClient)
 	if err != nil {
 		log.Fatal("service initialisation failed: " + err.Error())
 	}
 
+	// TODO: remove hardcoding of config.CLArgs.RegistryAddress value after the migration
+	// We cannot change the CL arguments for svc-api until migration is complete
+	config.CLArgs.RegistryAddress = "etcd:2379"
```

Comment:
Please remove the hard coding of registry address.

Reply:
updated

Reply:
Please check again, the code is still present